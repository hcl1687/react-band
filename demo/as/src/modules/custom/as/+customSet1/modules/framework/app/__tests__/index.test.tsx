import { mount, shallow } from 'enzyme'
import React from 'react'
import appFactory from '../index.entry'

const App = appFactory()

describe('custom/as/framework/app', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<App>hello</App>)
    expect(wrapper).toMatchSnapshot()
  })

  it('should has app class', () => {
    const wrapper = mount(
      <App>hello</App>
    )
    expect(wrapper.find('.app').length).toBe(1)
  })
})
