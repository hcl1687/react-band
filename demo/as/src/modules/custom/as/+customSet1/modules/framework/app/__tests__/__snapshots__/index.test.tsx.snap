// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`custom/as/framework/app should render correctly 1`] = `
<div
  className="app"
>
  hello
</div>
`;
